src/ontology/imports/dental_material_import.owl

```diff
@@ -460,7 +462,7 @@ updated defintion</obo:IAO_0000116>
     <!-- http://purl.obolibrary.org/obo/OHD_0000137 -->
 
     <Class rdf:about="http://purl.obolibrary.org/obo/OHD_0000137">
-        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/OHD_0000048"/>
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/imports/dental_material_import.owl#indirect"/>
```

Comment:
You need to revert this and keep the OHD IRI: `http://purl.obolibrary.org/obo/OHD_0000048`.  
Your classes will be added under the top level OHD classes.